src/Client.php

```diff
@@ -73,14 +73,17 @@ public function __construct(array $config = [])
         $this->configureDefaults($config);
     }
 
-    public function __call($method, $args)
+    public function __call($phpMethod, $args)
```

Comment:
this rename is unneeded, and should be reverted - the original name was better anywant